.eslintrc

```diff
@@ -18,26 +18,31 @@
     "platform_require": true
   },
   "plugins": [
+    "json",
     "mozilla",
     "react"
   ],
   "extends": [
     "eslint:recommended"
   ],
   "rules": {
-    "react/jsx-uses-react": 1,
-    "react/jsx-uses-vars": 1,
-
     "mozilla/components-imports": 1,
     "mozilla/import-globals-from": 1,
     "mozilla/this-top-level-scope": 1,
 
+    "react/jsx-uses-react": 1,
+    "react/jsx-uses-vars": 1,
+
     "array-bracket-spacing": [2, "never"],
+    "brace-style": 0,
```

Comment:
Actually, I should call this out too... I think this was set to `2` in JSCS, but it was throwing about 15 errors, mostly regarding inline JavaScript in react templates or an `else {...}` statement which had some inline comments above it. I disabled it because, like, whatever, man. Plus, adding `eslint-disable-line brace-style` wasn't working, and I had to do weird hacks like `/* eslint-disable-next-line brace-style */` and it was getting awkward and the code looked funny.